### DISCUSSION TITLE
Display the original field handle in the field layout editor

### DISCUSSION THREAD

**Discussion by MoritzLost** [craftcms/cms#14994] [Ideas]
The field layout editor makes it very difficult to see which field a particular field instance belongs to.

For example, I have multiple `Layout` fields with different options that are all used with the label `Layout` and handle `layout` in different entry types that are used for a content builder matrix field. Because the handle is overwritten, it's impossible to tell which field this field instance actually is:

![Screenshot 2024-05-15 at 12 05 37](https://github.com/craftcms/cms/assets/10146880/9d77c51c-f845-4c5b-887c-00fb8da73eb3)

Opening the field in a modal window doesn't help, I still don't see the original field handle:

![Screenshot 2024-05-15 at 12 08 31](https://github.com/craftcms/cms/assets/10146880/686bb3af-5d52-4fb9-a25a-0092ecbb7744)

The only way to see the original field handle is to remove the overwritten field handle so I can see the original field handle, which is used as a placeholder.

![Screenshot 2024-05-15 at 12 05 54](https://github.com/craftcms/cms/assets/10146880/c002014d-a683-4df2-9eac-0ce0e9fe83b8)

This is awkward and not very accessible.

## Proposed solution

Show the original field handle both in the field layout editor and in the modal window when editing a field.

Very quick mockup (`layout` is the instance handle and `layout_6_9` the original handle):

![Screenshot 2024-05-15 at 12 11 36](https://github.com/craftcms/cms/assets/10146880/f0a18b76-63de-4d7a-90d8-9c4fcbbf8f45)

**Comment by MattWilcox** [comment]
Seconding this, I'd love to see a "source field handle" for these, prominently displayed.

**Comment by brandonkelly** [comment]
> The only way to see the original field handle is to remove the overwritten field handle so I can see the original field handle, which is used as a placeholder.

You could also click the “Edit field” menu option, which opens the field’s global settings in a slideout.

**Reply by MoritzLost** [reply]
@brandonkelly Good point, thanks, that's a bit more convenient. Would still be useful to have to original field handle visible in the list of fields directly.
